addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -50,6 +50,15 @@ function factory(dependencies) {
             if (thread && notifyServer) {
                 thread.notifyFoldStateToServer('closed');
             }
+            if (this.env.device.isMobile) {
+                if (!this.env.messaging.discuss.isOpen) {
```

Comment:
group conditional expressions together in single conditional statement

Reply:
done